ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -318,6 +318,38 @@ def test_tensorflow_Atan(
     )
 
 
+# Atan2
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(set(ivy_tf.valid_float_dtypes))
+        ),
+        num_arrays=2,
+        shared_dtype=True,
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.Atan2"
+    ),
+)
+def test_tensorflow_Atan2(
+    dtype_and_x, as_variable, num_positional_args, fw, native_array
+):
+    dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
```

Comment:
you should treate with_out like the as_variable_flags and native_array_flags and put an argument for it in test_tensorflow_Atan2 like this:
def test_tensorflow_Atan2(
    dtype_and_x, as_variable, with_out, num_positional_args, fw, native_array
):
...
 helpers.test_frontend_function(
        input_dtypes=dtype,
        as_variable_flags=as_variable,
        with_out=with_out,
...
As it will also be handled automatically by the handle_cmd_line_args